doc/classes/Projection.xml

```diff
@@ -131,152 +151,194 @@
+			The first column of the [Projection].
 		</member>
 		<member name="y" type="Vector4" setter="" getter="" default="Vector4(0, 1, 0, 0)">
+			The second column of the [Projection].
 		</member>
 		<member name="z" type="Vector4" setter="" getter="" default="Vector4(0, 0, 1, 0)">
+			The third column of the [Projection].
 		</member>
 	</members>
 	<constants>
 		<constant name="PLANE_NEAR" value="0">
+			The index value of the projection's near clipping plane.
 		</constant>
 		<constant name="PLANE_FAR" value="1">
+			The index value of the projection's far clipping plane.
 		</constant>
 		<constant name="PLANE_LEFT" value="2">
+			The index value of the projection's left clipping plane.
 		</constant>
 		<constant name="PLANE_TOP" value="3">
+			The index value of the projection's top clipping plane.
 		</constant>
 		<constant name="PLANE_RIGHT" value="4">
+			The index value of the projection's right clipping plane.
 		</constant>
 		<constant name="PLANE_BOTTOM" value="5">
+			The index value of the projection bottom clipping plane.
 		</constant>
 		<constant name="IDENTITY" value="Projection(1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1)">
+			A [Projection] with no transformation defined. When applied to other data structures, no transformation is performed.
 		</constant>
 		<constant name="ZERO" value="Projection(0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0)">
+			A [Projection] with all values initialized to 0. When applied to other data structures, they will be zeroed.
 		</constant>
 	</constants>
 	<operators>
 		<operator name="operator !=">
 			<return type="bool" />
 			<param index="0" name="right" type="Projection" />
 			<description>
+				Returns [code]true[/code] if the projections are not equal.
+				[b]Note :[/b] Due to floating-point precision errors, this may return [code]true[/code], even if the projections are virtually equal. An [code]is_equal_approx[/code] method may be added in a future version of Godot.
 			</description>
 		</operator>
 		<operator name="operator *">
 			<return type="Projection" />
 			<param index="0" name="right" type="Projection" />
 			<description>
+				Returns a [Projection] that applies the combined transformations of this [Projection] and [param right].
 			</description>
 		</operator>
 		<operator name="operator *">
 			<return type="Vector4" />
 			<param index="0" name="right" type="Vector4" />
 			<description>
+				Projects (multiplies) the given [Vector4] by this [Projection] matrix.
 			</description>
 		</operator>
 		<operator name="operator ==">
 			<return type="bool" />
 			<param index="0" name="right" type="Projection" />
 			<description>
+				Returns [code]true[/code] if the projections are equal.
+				[b]Note :[/b] Due to floating-point precision errors, this may return [code]false[/code], even if the projections are virtually equal. An [code]is_equal_approx[/code] method may be added in a future version of Godot.
```

Comment:
```suggestion
				[b]Note:[/b] Due to floating-point precision errors, this may return [code]false[/code], even if the projections are virtually equal. An [code]is_equal_approx[/code] method may be added in a future version of Godot.
```

It should not have space after `Note` (same for other places).